f2/src/BusinessPage.js

```diff
@@ -25,7 +25,7 @@ export const BusinessPage = () => {
         <Page>
           <Layout columns={{ base: 4 / 4, md: 6 / 8, lg: 7 / 12 }}>
             <Stack spacing={10} shouldWrapChildren>
-              <BackButton route="/whatwasaffected">
+              <BackButton onClick={() => history.goBack()}>
                 <Trans id="businessInfoPage.backButton" />
```

Comment:
Are all of our BackButtons now the same? Ie do they all say “go back” and do a history.goBack()? If so, you could move both the onClick and the trans tag into the button.